contracts/extensions/CoinMachine.sol

```diff
@@ -53,6 +50,13 @@ contract CoinMachine is DSMath, ColonyExtension {
 
   uint256 emaIntake; // Averaged payment intake
 
+  // Modifiers
+
+  modifier onlyRoot() {
+    require(colony.hasUserRole(msg.sender, 1, ColonyDataTypes.ColonyRole.Root), "coin-machine-not-root");
```

Comment:
`coin-machine-caller-not-root` here as well, to match? I still don't understand why we've changed from `user` to `caller` though :sweat_smile:

Reply:
I'll make it user :)